site/styles/_variables.scss

```diff
@@ -46,6 +46,11 @@ $spacers: (
   '5':   ($spacer * 5),    // 50px
 );
 
+// Neutral Colors
+$grey: #999999;
```

Comment:
Can we choose a more semantic name? If a client requests a style change to say `#ff69b4`, then `$grey` doesn't makes as much sense. Or perhaps you really need `$grey` and some more explicit ones like `$form-input-bg-color`/`$form-input-border-color`.

Reply:
I'm sceptical about hard-coded 'neutral' colours... is this going to be one of those colours that needs changing from default for every site style because it never looks right or is never accessible with a client's chosen colours.  Maybe it would be better off using the body-bg-accent colour variable?

Reply:
I removed the neutral colors based on Jack's comment. It will definitely be safer using body-accent.